src/host/README.md

```diff
@@ -0,0 +1,122 @@
+# src/host
+
+This directory implements the systems used by the `bin/host.js` command line tool.
+
+The `bin/host.js` tool has two commands. The main command operates a server and `bin/agent.js` instance to run test plans and the `plan` subcommand reads files from disk and packages them for the main command.
+
+A short invocation of the main command can be made with a list of files or file globs that make up a test plan. A test plan should contain tests for a single assistive technology to be tested. If for example all the reference files and test files are in two directories this command could be executed.
+
+```sh
+$ bin/host.js reference/** at/** >result.json
+Starting...
+Reference server listening on 'http://localhost:52147'.
+Reference available on 'http://localhost:52147/gtmoyv'.
+Removing reference from 'http://localhost:52147/gtmoyv'.
+Stopping...
+```
+
+Developing `bin/host.js` you may want to use the mock test runner in `bin/agent.js`. Calling `host.js` with `--agent-arg=--mock` will enable the mock test runner.
+
+```sh
+$ bin/host.js --agent-arg=--mock reference/** at/** >result.json
+...
+```
+
+An advanced invocation combines the plan subcommand and main command by piping the output of multiple `bin/host.js plan` calls to the main command. The output will be a stream of json objects, one for each run plan.
+
+```sh
+$ plans=(
+array> breadcrumb
+array> checkbox
+array> )
```

Comment:
In basic bash it would render as

```sh
$ plans=(
> breadcrumb
> checkbox
> )
```

I had included the shell prompts which in `()` lists starts with `>` normally. My shell configuration had changed that `array>`.